src/ol/worker/webgl.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @module ol/worker/webgl
+ * A worker that does cpu-heavy tasks related to webgl rendering
```

Comment:
Minor, but the description needs to go above the `@module` annotation.  See #9563.

Reply:
fixed